### ISSUE TITLE
Typo in package name

### ISSUE THREAD

**Issue by PallHaraldsson** [Cvikli/Aritmetics.jl#1]
I'm not sure it's intentional, it's ok in the README.

Before you register, or think about renaming, is it a good name?

It's a very general name (when correct), so I'm not sure it will be accepted, if you try to register. Maybe you didn't have that in mind?

Apparently without the h is always wrong in English, only ok in a few languages like Romanian and Albanian (and FYI: it's "reikningur" in Icelandic...):

https://en.wiktionary.org/wiki/arithmetic

>Alternative forms arsmetrike, arsmetryke, arithmetike (obsolete)

**Comment by Cvikli** [comment]
OMG! You have it right. Sorry for that. So. What if I go with Arithmetics.jl ? (I also saw it should be plural, so I guess this should also work. 
Also thank you for commenting! 

Of course I have it in mind although I know it isn't sure it will be accepted as that topic is extremely wide. 
So Maybe I will add that into the README.md, if someone wants to extend this or take the repository place I can give this to him or I can accept his arithmetic extension requests.

**Comment by Cvikli** [comment]
OKay: Created the new one: https://github.com/Cvikli/Arithmetics.jl

Link corrected hopefully.

**Comment by PallHaraldsson** [comment]
It seems you changed the typo there, i.e. I confirmed for the module name at least. The repo is being indexed, and it seems I wouldn't see a commit change anyway.

At least you didn't want to intend a typo... We'll see about at registration, I'm not saying I would object to the general name, I just took a quick look at the package. FYI: I've done some activation functions, and know where others are and derivatives, in e.g. BetaML, and another Flux related package.
